go.mod

```diff
@@ -138,7 +138,7 @@ require (
 	golang.org/x/text v0.14.0 // indirect
 	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/genproto/googleapis/api v0.0.0-20230913181813-007df8e322eb // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20231120223509-83a465c0220f // indirect
```

Comment:
Wondering if we could (should?) somehow implement a check for all the `google.golang.org/genproto/googleapis/*` dependencies to be aligned, I don't know how much relation they have, but they all come from the same repository (and don't provide tagged releases), but wondering if it'd be good to have them all at the same snapshot (when they were regenerated).

Reply:
(sorry; random comment on this PR)